minio/api.py

```diff
@@ -795,6 +802,15 @@ def put_object(self, bucket_name, object_name, data, length,
         if length > MAX_MULTIPART_OBJECT_SIZE:
             raise InvalidArgumentError('Input content size is bigger '
                                        ' than allowed maximum of 5TiB.')
+
+        if part_size < MIN_PART_SIZE:
+            raise InvalidArgumentError('Input part size is smaller '
+                                       ' than allowed minimum of 5MiB.')
+
+        if part_size > MAX_PART_SIZE:
+            raise InvalidArgumentError('Input part size is bigger '
+                                       ' than allowed maximum of 5GiB.')
+
```

Comment:
Should there also be a check for `length` < `part_size*10000` ?

Reply:
Yes definitely @kannappanr